docs/memmap.tex

```diff
@@ -11,10 +11,11 @@ \chapter{Memory Map}
 \item[\$200 - \$258] Text input buffer.
 \item[\$33C - \$3FB] PAD Scratch pad memory, Cassette Buffer, untouched by DurexForth. 
 \item[\ldots]
-\item[\$801 - \texttt{here}] Forth Kernel followed by dictionary.
+\item[\$801 - \texttt{here}] Forth Kernel followed by code and data space.
 \item[\ldots]
 \item[\texttt{bufstart} - \texttt{eof}] Editor text buffer. Grows upwards as needed. Default \texttt{bufstart} is \$7000.
 \item[\ldots]
+\item[\texttt{latest @} - \$9fff] Dictionary. Grows downwards as needed.
```

Comment:
Looks fine to me.
I wonder where editor buffer should go in the long term. I was wondering if it's possible to move hires graphics so that it sits beneath Basic ROM, then $e000-$ffff could be used for editor buffer. But it's a tricky question and maybe no great answers.

Reply:
`HERE $2000 ALLOC 1+ bufstart !`

I mean, does it have to be a fixed address?

Reply:
:)  That's a good point. I guess the problem with putting it inside the word space has been that once you get a buffer overrun, it would crash pretty fast.

Reply:
One more sentinel won't break the bank. ;)